source/MaterialXFormat/File.cpp

```diff
@@ -141,6 +148,32 @@ bool FilePath::exists() const
 #endif
 }
 
+FilePath FilePath::normalize() const
```

Comment:
This naming convention looks good to me, but perhaps we should name the method "getNormalized" as we do for vectors, making it clear that the original FilePath is left unmodified.